fastlane/Fastfile

```diff
@@ -1114,25 +1036,80 @@ inal copy, and try again.")
     )
   end
 
-  def create_merge_release_branch
-    Fastlane::Helper::GitHelper.create_branch(merge_branch_name, from: release_branch_name)
+  # - Loop though all PRs which were still open and were assigned the milestones of the code-frozen version,
+  #   and move them to the next milestone + leaving a PR comment about it
+  # - Add the ❄️ frozen marker to the milestone being frozen
+  def update_milestone()
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/689cde6eef99b303ccc74cf419b2f1b53392b113/fastlane/Fastfile#L1042"></span>Style/DefWithParentheses: Omit the parentheses in defs when the method doesn't accept any arguments.</td>
    </tr>
  </tbody>
</table>